### ISSUE TITLE
ID reseting bug

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5640]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
There's a bug where IDs can randomly reset, tonight for example we had our weekly icc 25hc run and we cleared 10/12 on hc then we had a break before PP and ID just reseted and we had to redo, this happened before with other raids randomly reseting few days usually after doing them, most often I had voa reseting out of nowhere, heard many ppl having this issue too.
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by MalkaviaDW** [comment]
Hello there, 

Thanks for your report, we were already looking into it.

Cheers!

**Comment by Byssarod** [comment]
Perhaps it have something to do with changing raid leader. Example: Character CCCC is raid leader. Few raid members enter instance and start clearing trash. Character CCCC then give raid leader to someone who isnt in raid instance yet and then logout/ disconnect.
